app/models/user.rb

```diff
@@ -33,6 +33,7 @@ class User < ApplicationRecord
   has_many :requests, dependent: :destroy
   has_many :stores, dependent: :destroy
   has_many :items, through: :stores # must come after has_many :stores declaration
+  has_many :number_log_entries, through: :logs
```

Comment:
I don't think we ended up using this, but I think we might as well add it, for symmetry with `text_log_entries`.